src/city-council-district/city-council-district.service.spec.ts

```diff
@@ -23,10 +26,31 @@ describe("City Council District service unit", () => {
     );
   });
 
-  it("service should return a findCityCouncilDistrictsQueryResponseSchema compliant object", async () => {
-    const cityCouncilDistricts = await cityCouncilDistrictService.findMany();
-    expect(() =>
-      findCityCouncilDistrictsQueryResponseSchema.parse(cityCouncilDistricts),
-    ).not.toThrow();
+  describe("findMany", () => {
+    it("service should return a findCityCouncilDistrictsQueryResponseSchema compliant object", async () => {
+      const cityCouncilDistricts = await cityCouncilDistrictService.findMany();
+
+      expect(() =>
+        findCityCouncilDistrictsQueryResponseSchema.parse(cityCouncilDistricts),
+      ).not.toThrow();
+    });
+  });
+
+  describe("findCapitalProjectsByCityCouncilId", () => {
```

Comment:
please add cases for
- [default limit and offset](https://github.com/NYCPlanning/ae-zoning-api/blob/f5a4145099d3c8a0585a5478d5b15a8b9bca2608/src/tax-lot/tax-lot.service.spec.ts#L31)
- [should return a tax lot list, using user specified limit and offset](https://github.com/NYCPlanning/ae-zoning-api/blob/f5a4145099d3c8a0585a5478d5b15a8b9bca2608/src/tax-lot/tax-lot.service.spec.ts#L43)